composites/OnboardingWizard/config/production-config.js

```diff
@@ -4,364 +4,60 @@ import PostTypeVisibility from "../components/custom_components/PostTypeVisibili
 
 let configuration = {
-	'finishUrl' : '',
-	"endpoint": "http://127.0.0.1:8882/onboarding?wp_nonce=nonce",
-	"customComponents": {
-		MailchimpSignup, PublishingEntity, PostTypeVisibility, ConnectGoogleSearchConsole,
-	},
 	"fields": {
-		"upsellConfigurationService": {
+		"welcomeService": {
 			"componentName": "HTML",
-			"properties": {
-				"html": "You can now have Yoast configure Yoast SEO for you!",
-			},
-		},
-		"upsellSiteReview": {
-			"componentName": "HTML",
-			"properties": {
-				"html": "Get more visitors! Our SEO website review will tell you what to improve!",
-			},
+			"properties": { "html": "<p>Welcome to the Yoast Local SEO configuration wizard. In a few simple steps we'll help you configure your Local SEO settings to match your website's needs!<\/p>" }
 		},
-		"successMessage": {
-			"componentName": "HTML",
-			"properties": {
-				"html": "Good Job! You've finished setting up Yoast SEO. Thereby you've covered the technical part of your site's SEO. Now it's time to focus on optimizing your content for onpage SEO. You can use our content analysis for that: <br>{gif_showing_content_analysis}",
-			},
-		},
-		"mailchimpSignup": {
-			"componentName": "MailchimpSignup",
-			"properties": {
-				"label": "If you would like us to keep you up-to-date regarding Yoast SEO and other plugins by Yoast, subscribe to our newsletter:",
-				"mailchimpActionUrl": "{http://yoast.us1.list-manage1.com/subscribe/post?u=ffa93edfe21752c921f860358&amp;id=972f1c9122}",
-				"currentUserEmail": "{current_user_email:String}",
-			},
-			"data": "false",
-		},
-		"environment": {
-			"componentName": "Choice",
-			"properties": {
-				"label": "Please specify the environment {site_url} is running in.",
-				"choices": {
-					"production": {
-						"label": "Production - live site.",
-					},
-					"staging": {
-						"label": "Staging - copy of live site used for testing purposes only.",
-					},
-					"development": {
-						"label": "Development - locally running site used for development purposes.",
-					},
-				},
-			},
-			"data": "development",
-			"default": "production",
-		},
-		"siteType": {
+		"singleOrMultiple": {
 			"componentName": "Choice",
 			"properties": {
-				"label": "What type of site is {site_url}?",
 				"choices": {
-					"blog": {
-						"label": "Blog",
-					},
-					"shop": {
-						"label": "Shop",
-					},
-					"news": {
-						"label": "News site",
-					},
-					"smallBusiness": {
-						"label": "Small business site",
-					},
-					"corporateOther": {
-						"label": "Other corporate site",
-					},
-					"personalOther": {
-						"label": "Other personal site",
-					},
+					"single": { "label": "One single location" },
+					"multiple": { "label": "Multiple locations" }
 				},
+				"label": "Please specify if you will be using one single location or multiple locations (Yoast SEO will create a new Custom Post Type from you where you can manage your locations)."
```

Comment:
Can we change the text to `Please specify if you will be using one single location or multiple locations. Yoast SEO will create a new Custom Post Type from you where you can manage your locations.` ?
This way we don't have the parenthesis in the sentence that makes it very long.